src/footer/footer.scss

```diff
@@ -44,8 +44,13 @@
 
       > a {
         @include iui-anchor;
-        @include themed {
-          color: t(iui-text-color-muted);
+        @include iui-focus;
+
+        &,
+        &:hover {
```

Comment:
Um, why do we not want any hover state on the links?

Reply:
The hyperlinks do gain an underline on hover, but this is how they were styled prior to this bug.  I could introduce a `$iui-text-color-muted-overlay` similar to how we have in other places if you'd like to see a color change on hover as well.

Reply:
Oh I was looking at the wrong image. Yeah I guess the underline is fine if that's how it was before.